### PR TITLE
initial check in  for storage module for Azure Stack

### PR DESCRIPTION
**Pull request by erich-wang**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by blueww** [comment]
@erich-wang 
Would you please give some background why we do this change?
Will this change impact master branch?

File: src/Storage/Storage.Management/Az.Storage.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.9.0'
```
Comment (wyunchi-ms) [inline src/Storage/Storage.Management/Az.Storage.psd1]: Isn't the module version be `0.0.100`?

**Comment by erich-wang** [comment]
> @erich-wang
> Would you please give some background why we do this change?
> Will this change impact master branch?

@blueww , thank you for your comments. This branch is for Azure Stack and won't be merged back to master.

**Pull request merged**